src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
+$number-of-columns: 12;
+
+@mixin common {
+  display: flex;
+  justify-content: flex-start;
+  width: calc(100% + #{$gutters});
+  margin-left: calc(#{$gutters} / -2);
+}
+
+.ec-grid {
+  overflow: hidden;
+  width: 100%;
+}
+
+@for $i from 1 through $number-of-columns {
+  .ec-col-#{$i} {
+    margin: calc(#{$gutters} / 2);
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: calc((100% * ( #{$i} / #{$number-of-columns})) - #{$gutters});
+
+    // For IE11 remove extra 4 px in case we have borders
+    @media all and (-ms-high-contrast: none) {
```

Comment:
Do you have any link where this problem is shown? I'm curious to know why `-ms-high-contrast` was used as a detection?

Reply:
1. -ms-high-contrast is so i can target only IE11 with css. Rest of the browsers will ignore.
2. IE11 seems to ignore border-box when flex. I remove 4px which seems to be the max px we have for tfo (each border 1px)

Reply:
I'm not sure I follow, what borders are problem? There are no borders in the grid, only in the story. In case that there is no border on the container, these 4 pixels will be missing.